chapters/curate-maintain/content/reading/README.md

```diff
@@ -1,25 +1,35 @@
 
-the central part is content management, around a repository
+It is important for such a resource to clearly identify who the maintainer is, for the purpose of transparency and good communication.
 
-typically Git & GitHub
+The content is typically organized in a `Git` repository, which has a series of advantages like version control, branching, history, etc.
+It is also recommended to use a managed development platform like `GitHub` or `GitLab`, which is built on top of `Git`, while also providing additional useful features, like code review, issues, pull requests, integrations, and others.
 
-versioning, maintaining up-to-date
+The content should be published under a license.
+An example of a widely used license for public education resources is `Creative Commons`.
 
-encouraging contributions, integrating changes
+#### Contributions
 
-creating issues, monitoring issues and pull requests
+The content maintainer should foster a welcoming environment for outside contributions.
+This can be achieved by:
+- establishing clear contribution guidelines (for example in a `CONTRIBUTING.md` file)
+- labeling issues with tags such as `help wanted` or `good first issue`, to help newcomers find tasks to contribute
+- acknowledging the work of contributors (for example in the repository `README.md` file or in any other suitable place)
 
-licensing, ownership
+There should also be an established process for handling contributions whenever they arrive.
+Some steps that the maintainer should take are:
+- review the contribution as soon as possible.
+Depending on the team size, this can be done by the maintainer, or it could be delegated to other members of the content development team.
+Tools like `GitHub` have features that can help with this: an issue can have a person designated as assignee, while a pull request can have one or multiple reviewers.
+- engage in a constructive discussion with the contributor to clarify any questions or concerns.
+Provide feedback and guidance for improvement if the contribution needs changes.
+- ensure that the contribution follows the style guidelines.
+This can be partially automated using linters and automated checkers, together with features like `GitHub Actions`.
+- if applicable, test the contribution to ensure it works as intended.
+- thank the contributor for their effort.
 
-the process of creating a repository from zero
-how the structure should be created;
-initial contributions;
-submission via PRs and review process
+#### Creating a repository
 
-the process of improving a repository via contributions;
-how the contribution + review process should happen;
-steps to be done, reviewers, assignees
+If a new content repository needs to be created from scratch, a good starting point is to clone the [oer-template](https://github.com/open-education-hub/oer-template) repository.
```

Comment:
I would add a short phrase on what the `oer-template` contains. Just say that it has a basic repository structure with all the tools included, and any kind of content can be added on top of that.